__tests__/hooks/useByteFormatter.js

```diff
@@ -0,0 +1,30 @@
+import { timeouts } from '../constants.json';
+
```

Comment:
out of curiosity, what languages use a different file size system? I believe Arabic does...

Reply:
AFAIK, all of them are kB/MB/GB.